src/locale/misc/views/en.yml

```diff
@@ -14,6 +14,10 @@ columnDialog:
     tag:
       description: Show a checkmark if the person has this tag
       title: Tag
+    toggle:
+      columnTitle: Toggle
+      description: Checkbox column
+      title: Toggle
```

Comment:
I am curious about the reasoning behind having both title and columnTitle. Is it based on a prediction that we will come to have choices where we want the column default title to be titled something different than the choice title?

Reply:
Yes! That was my request, because I don't think the choice should be called "Toggle", but something more communicative that we can come up with once all the choices are done and we do a full review of all the texts in the dialog.